inc/cachify_db.class.php

```diff
@@ -131,7 +132,7 @@ public static function print_cache( $cache ) {
 
 		/* Signature */
 		if ( isset( $cache['meta'] ) ) {
-			echo self::_cache_signature( $cache['meta'] );
+			echo self::_cache_signature( $sigDetail, $cache['meta'] );
```

Comment:
The DB cache signature can provide additional information only available at runtime (e.g. number of actual DB queries). There might be any reason why one would like to see this...

Only possibility to unify call of the "basic" signature would be to add a second comment at runtime or drop the additional data completely (remove `$sig_detail`).